app/src/main/res/layout-land/fragment_billing_address.xml

```diff
@@ -0,0 +1,323 @@
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@id/mid_guideline"
+        app:layout_constraintTop_toBottomOf="@id/top_separator"
+        />
+    <include
+        android:id="@+id/bonus_layout"
+        layout="@layout/view_purchase_bonus"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/big_margin"
+        android:layout_marginTop="19dp"
+        android:layout_marginEnd="@dimen/big_margin"
+        android:visibility="invisible"
+        app:layout_constraintEnd_toEndOf="@id/mid_guideline"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/payment_methods_header"
+        tools:visibility="visible"
+        />
+
+    <TextView
+        android:id="@+id/bonus_msg"
+        style="@style/TopUpTextStyle.Small.BonusText"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/big_margin"
+        android:layout_marginTop="11dp"
+        android:layout_marginEnd="@dimen/big_margin"
+        android:layout_marginBottom="11dp"
+        android:gravity="center_vertical"
+        android:text="@string/gamification_purchase_body"
+        android:visibility="invisible"
+        app:layout_constraintEnd_toEndOf="@id/mid_guideline"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/bonus_layout"
+        />
+
+
+    <TextView
+        android:id="@+id/title"
+        style="@style/BillingAddress.Title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/big_margin"
+        android:layout_marginTop="@dimen/big_margin"
+        android:text="@string/purchase_address_title"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintStart_toEndOf="@id/mid_guideline"
+        app:layout_constraintTop_toBottomOf="@id/top_separator"
+        />
+
+    <ScrollView
+        android:id="@+id/info_view"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginTop="@dimen/normal_margin"
+        android:overScrollMode="never"
+        android:scrollbars="none"
+        app:layout_constraintBottom_toTopOf="@id/bottom_separator"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@id/mid_guideline"
+        app:layout_constraintTop_toBottomOf="@id/title"
+        >
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+          android:id="@+id/content_main"
+          android:layout_width="match_parent"
+          android:layout_height="216dp"
+          android:layout_marginStart="@dimen/big_margin"
+          android:layout_marginEnd="@dimen/big_margin"
```

Comment:
Can't this layout be extracted so that can be used as an include in both payment and topup flow?